src/components/Map/PlantingSiteMap.tsx

```diff
@@ -182,22 +192,10 @@ export default function PlantingSiteMap(props: PlantingSiteMapProps): JSX.Elemen
   }
 
   return (
-    <Box sx={{ display: 'flex', flexGrow: 1 }}>
+    <Box sx={{ display: 'flex', flexGrow: 1, position: 'relative', minHeight: '436px' }}>
```

Comment:
`position: 'relative'` added here to make the element "positioned." This allows the absolutely positioned child elements to be properly positioned

Reply:
`minHeight` added because a large control obscures much of the map area. Got this height from the tracking mobile figma

Reply:
wondering if chudo can reduce the padding on the slider? it is rather large i agree

Reply:
left a comment on the Figma asking about this